From 3abc26f07ebb7779fd5cb00092829f437ebc71c2 Mon Sep 17 00:00:00 2001
From: Jan Synacek <jsynacek@redhat.com>
Date: Tue, 11 Sep 2018 13:18:27 +0200
Subject: [PATCH 2/2] gnutls.el: increase gnutls-min-prime-bits to 1024 by
 default

Get rid of the gnutls warning saying that the DH key has been lowered.

Related: #1403643
---
 lisp/net/gnutls.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/net/gnutls.el b/lisp/net/gnutls.el
index 2585f48..ce69244 100644
--- a/lisp/net/gnutls.el
+++ b/lisp/net/gnutls.el
@@ -79,7 +79,7 @@ The files may not exist, in which case they will be ignored."
                  (repeat (file :tag "Bundle filename"))))
 
 ;;;###autoload
-(defcustom gnutls-min-prime-bits 256
+(defcustom gnutls-min-prime-bits 1024
   ;; Several mail servers send fewer bits than the GnuTLS default.
   ;; Currently, 256 appears to be a reasonable choice (Bug#11267).
   "Minimum number of prime bits accepted by GnuTLS for key exchange.

